docs/content/docs/developing.md

```diff
@@ -4,7 +4,7 @@ description = "Contributing to kube-profefe"
 weight = 30
 draft = false
 toc = true
-bref = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur neque risus, maximus malesuada massa eget, auctor vulputate ipsum. Suspendisse magna libero, iaculis non ante nec, laoreet faucibus ligula. Suspendisse sed enim urna. Quisque libero dui, porttitor eu diam quis, elementum mattis nunc."
+bref = ""

```

Comment:
That's better than the previous one but I think we can do a bit better, do you mind do add a good description? Thanks a lot

Reply:
Updated with description